src/ui/window/sdl/TopWindow.cpp

```
@@ -30,6 +31,101 @@
+#else
+  return true;
+#endif
+}
+
+/**
+ * Count the fingers currently touching @p touch_id.  Depending on the
+ * SDL version, the finger that triggered a SDL_FINGERUP event may still
+ * be listed; pass its id in @p lifted to exclude it.
+ */
+[[gnu::pure]]
+static unsigned
```

Cppcheck (2.21.0)

[error] Syntax error, token has no link.

(internalAstError)